source/type/message.h

```diff
@@ -129,6 +129,21 @@ inline std::string to_string(ChannelType ct) {
     }
 }
 
+struct Property {
```

Comment:
I'd rather have `Disruption`Property

Reply:
As discussed yesterday, we won't change this one.